test/test_integration.py

```diff
@@ -1184,6 +1185,54 @@ def on_click(n_clicks):
         time.sleep(1)
         self.snapshot("Tabs 2 rendered ")
 
+        # do some extra tests while we're here
+        # and have access to Graph and plotly.js
+        self.check_graph_config_shape()
+        self.check_plotlyjs()
+
+    def check_plotlyjs(self):
+        # find plotly.js files in the dist folder, check that there's only one
+        all_dist = os.listdir(dcc.__path__[0])
+        js_re = '^plotly-(.*)\.min\.js$'
+        plotlyjs_dist = [fn for fn in all_dist if re.match(js_re, fn)]
+
+        self.assertEqual(len(plotlyjs_dist), 1)
+
+        # check that the version matches what we see in the page
+        page_version = self.driver.execute_script('return Plotly.version;')
+        dist_version = re.match(js_re, plotlyjs_dist[0]).groups()[0]
+        self.assertEqual(page_version, dist_version)
+
+    def check_graph_config_shape(self):
+        config_schema = self.driver.execute_script(
+            'return Plotly.PlotSchema.get().config;'
+        )
+        with open(os.path.join(dcc.__path__[0], 'metadata.json')) as meta:
+            graph_meta = json.load(meta)['src/components/Graph.react.js']
+            config_prop_shape = graph_meta['props']['config']['type']['value']
```

Comment:
kind of ugly to have to use `metadata.json` for this... I was hoping I could get this directly from the `Graph` component, but `PropTypes` seems to turn everything into a validation function, I don't see a way to get the input params back out.